lib/src/firebase_performance_dio.dart

```diff
@@ -16,12 +16,14 @@ class DioFirebasePerformanceInterceptor extends Interceptor {
   DioFirebasePerformanceInterceptor({
     this.requestContentLengthMethod = defaultRequestContentLength,
     this.responseContentLengthMethod = defaultResponseContentLength,
+    this.generateKey,
```

Comment:
Where do we use this property ?

Reply:
this provide to the user option to generate key on his way. If it is null, the default implementationwill be used.

Reply:
Do we really need this ?